docs/provider-configuration.md

```diff
@@ -179,8 +179,21 @@ These configuration options for the OpenStack provider pertain to the load
 balancer and should appear in the `[LoadBalancer]` section of the `$CLOUD_CONFIG`
 file.
 
+Although the openstack-cloud-controller-manager was initially implemented with
+Neutron-LBaaS support, Octavia is recommended now because Neutron-LBaaS has
+been deprecated since Queens OpenStack release cycle and not longer accepted
+new feature enhancements. As a result, lots of advanced features in
+openstack-cloud-controller-manager rely on Octavia, even the CI is running
+based on Octavia enabled OpenStack environment. Functionalities are not
+guaranteed if using Neutron-LBaaS.
+
 ##### Load Balancer Optional Parameters
 
+* `use-octavia`: Used to determine whether to look for and use an
+  Octavia LBaaS V2 service catalog endpoint. Valid values are `true` or `false`.
+  Where `true` is specified and an Octavia LBaaS V2 entry can not be found, the
+  provider will fall back and attempt to find a Neutron LBaaS V2 endpoint
+  instead. The default value is `false`.
```

Comment:
@lingxiankong so by default code uses neutron Lbaas? shouldnt we change to default to octavia as the prior one is deprecated?

Reply:
Yeah, sounds good to me, we are still getting some that related to Neutron-LBaaS which is annoying. I will submit a patchset with that information.

Reply:
+1 to default to Octavia
and just to confirm (not sure ) we don't have anything that only on neutron-lbaas but not on octavia, correct?

Reply:
No, we don't

Reply:
ok, +1 to me ,thanks